snuba/admin/yarn.lock

```diff
@@ -2202,7 +2202,7 @@ brace-expansion@^2.0.1:
   dependencies:
     balanced-match "^1.0.0"
 
-braces@^3.0.3:
+braces@^3.0.1, braces@^3.0.2, braces@^3.0.3, braces@~3.0.2:
```

Comment:
There is apparently a [vulnerability](https://github.com/getsentry/snuba/security/dependabot/53) in braces versions older than `3.0.3`. Does this line bring in older versions of braces with vulnerabilities?